checker/dune

```diff
@@ -16,6 +16,11 @@
  (wrapped true)
  (libraries coq.lib))
 
+(install
+ (section bin)
+ (package coq)
+ (files (main.bc as coqchk.byte)))
```

Comment:
Is this useful?

Reply:
I added it for completeness, I dunno.

Reply:
I think remove this, wait a bit for anyone else interested to speak up, then I'll merge.